### PR TITLE
Add `docs/ossm` folder

### PR DESCRIPTION
**Pull request by nrfox**
#### What type of PR is this?

- [ ] Enhancement / New Feature
- [ ] Bug Fix
- [ ] Refactor
- [ ] Optimization
- [ ] Test
- [x] Documentation Update

#### What this PR does / why we need it:

Adds a `docs/ossm` folder for ossm specific content.

#### Which issue(s) this PR fixes:

Fixes #

Related Issue/PR #

### CODE REVIEW

**Review by sridhargaddam — APPROVED** [review]

File: docs/ossm/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Return to Sail Operator documentation](../)
+
+# User Documentation
+
+This documentation is specific to the OpenShift Service Mesh product and may differ slightly from the [Sail Operator documentation](../README.md). For OpenShift Service Mesh 3, this documentation should be prioritized over the [Sail Operator documentation](../README.md). However, the official source of truth is the product documentation available at [docs.redhat.com](https://docs.redhat.com). If a particular topic is not covered here, you can refer to the [Sail Operator documentation](../README.md) or [Istio.io](https://istio.io) documentation as supplementary resources.
```
Comment (sridhargaddam) [inline docs/ossm/README.md:5]:
```suggestion
This documentation is specific to the OpenShift Service Mesh product and may differ slightly from the [Sail Operator documentation](../README.md). For OpenShift Service Mesh 3, this documentation should be prioritized over the [Sail Operator documentation](../README.md). However, the official source of truth is the product documentation available at [docs.redhat.com](https://docs.openshift.com). If a particular topic is not covered here, you can refer to the [Sail Operator documentation](../README.md) or [Istio.io](https://istio.io) documentation as supplementary resources.
```

**Review by sridhargaddam — CHANGES_REQUESTED** [review]
Lets use https://docs.openshift.com/ instead of docs.redhat.com

File: docs/ossm/README.md (hunk 1 above)
Comment (sridhargaddam) [reply docs/ossm/README.md:5]: Just noticed the comment from Gwynne regarding the decomissioning of the url. Approving the PR. Thanks Nick.

**Review by sridhargaddam — APPROVED** [review]

**Review by fjglira — APPROVED** [review]

**Pull request merged**